ttnn/cpp/ttnn/operations/ccl/line_all_gather/device/line_all_gather_op.cpp

```diff
@@ -90,6 +90,30 @@ Tensor line_all_gather(
             const std::vector<std::optional<const Tensor>>& optional_input_tensors,
             const std::vector<std::optional<Tensor>>& optional_output_tensors) mutable -> std::vector<Tensor> {
 
+            std::cout << "YIKES YIKES YIKES YIKES YIKES YIKES YIKES YIKES" << std::endl;
```

Comment:
YIKES, let's remove this 😂

Reply:
Yea sorry the changes were good to go then I added more changes to enable a new mode and do some debug. I'll pull this back into draft while I do the general cleanup